### ISSUE TITLE
Add an Accessibility Statement to the Documentation Page

### ISSUE THREAD

**Issue by mgifford** [11ty/eleventy#1364]
This one is pretty good https://www.gatsbyjs.com/accessibility-statement/

Lots of other examples out there though.  

Useful to demonstrate that this project cares about accessibility and is doing something about it.

**Comment by binyamin** [comment]
@mgifford Thanks for the suggestion.

Here are a couple of thoughts off the top of my head:
- I don't think it's fair to pick a single under-represented group to care about. Would a general, non-specific "inclusive policy" page serve this need?
- I agree that accessibility is important; however, is this the best medium? Maybe some text on the front page saying"AA WCAG compliant" would have a wider reach.

**Comment by mgifford** [comment]
* Thanks
* No. Couple things on this point.
  - It's not about picking favourites, however it is worth while to consider that disabilities cut across all of society. If we are lucky enough to grow old, we will eventually have disabilities. It's just biology.
  - There are many patterns on the web that exclude about 1/5 people. Plus, even if people don't have permanent disabilities there are temporary & situational disabilities which affect everyone. This isn't the case with other inclusion issues. 
  - Yes, there should be more women, people of color, trans.... people in tech communities. But they aren't limited by the technology. 
  - There are local, regional, national and international laws that address accessibility. There are W3C guidelines and ISO standards. So really it is an odd comparison when looking at other important inclusion issues. 
* Text saying that the website or technology is compliant was popular in the 90's.  It didn't work, nor will it. Your site and software doesn't meet WCAG 2.0 AA, and in many places in the world the regulations are for WCAG 2.1 AA.  Soon jurisdictions will be required to meet WCAG 2.2 AA and WCAG 3.0 is being actively developed. 

Some examples in Europe:
https://gds.blog.gov.uk/2019/05/21/accessibility-update-sample-accessibility-statement-monitoring-and-enforcement/

From some other popular software:
* https://www.drupal.org/about/features/accessibility
* https://wordpress.org/about/accessibility/
* https://www.joomla.org/accessibility-statement.html
* https://reactjs.org/docs/accessibility.html

**Comment by binyamin** [comment]
This suggestion came off the top of my head. I'm sorry you didn't like it. Please take some time to calm down, so we can have a constructive conversation.

> Your site and software doesn't meet WCAG 2.0 AA

Thanks for bringing this to our attention. Feel free to create a new issue.

**Comment by mgifford** [comment]
No worries..  Just wanted to address your points..  I'm happy to discuss any of this. I'm definitely not upset, just got a reasonable amount of experience with this topic. 

Also, I haven't actually tested the site, but am making some assumptions based on the fact that 99% of the top million sites don't come close to meeting it.

**Comment by binyamin** [comment]
Thanks for clarifying. Fyi, I've been working this summer in a camp for children with disabilities. I also try to make all my websites accessible.

As for the 11ty site, I know @zachleat makes a big deal of getting perfect lighthouse scores, which includes a11y tests.

Also, I don't think a11y would apply to the 11ty software. As far as I know, the only html not provided by the user is the html rendered from markdown by `markdown-it`. So if there's anything wrong, it is likely an upstream issue.

**Comment by binyamin** [comment]
Either way, I don't think I should be making the call here.

**Comment by mgifford** [comment]
Thanks for the explanation @binyamin.  I did notice the accessibility leaderboard was tied to Google Lighthouse. I thought that was a pretty innovative way to encourage that sites follow best practices.  I don't know @zachleat but it's a nice start.

Google Lighthouse uses the axe accessibility engine for it's tests just like AccessibilityInsights.io and many others. It's a great open source tool that GatsbyJS had integrated into the command line.  

Still these automated tools catch only about 1/3rd of the issues covered by WCAG 2.0 AA and they will never reach 100% coverage.  

Building automated testing (like axe) into the build process might be helpful.  More likely though it is just in making sure that your own documentation is accessible and that your examples are following best practices.  

Could be a pretty simple issue.

**Comment by zachleat** [comment]
Love this idea. I’ll work on it.

**Comment by zachleat** [comment]
@mgifford I’m also very interested to know more about where our “site and software doesn't meet WCAG 2.0 AA”—can you elaborate? I’d love to fix where we’re failing

**Comment by mgifford** [comment]
I don't know @zachleat I haven't had the time to investigate.

Gatsby integrates axe automated testing into the build process so it stands out as one interface that have gotten more accessible over time https://www.gatsbyjs.com/blog/2020-06-29-gatsby-most-accessible-webaim-million/

It could be just reviewing examples and documentation to see that you are following best practices.  

Even just stating what you've done to meet basic due diligence on accessibility can be useful.  Main site looks ok in the WAVE toolbar.  Have you done more than that?

**Comment by zachleat** [comment]
Oh oh, right on—I thought you had noticed a specific issue and just wanted more info if that was the case. Totally on board with this—I’m working on it as we speak.

**Comment by mgifford** [comment]
Partly it's just a matter of saying "we care about this, we've put some thought into it, but if you find ways to make it more inclusive, please let us know.  We're a community that wants to involve everyone!"

**Comment by zachleat** [comment]
First draft is https://github.com/11ty/11ty-website/commit/41b1dce9def8912f0865169539fb1d7675b5a1d7#diff-9b50937d29d94fa150f09761d4bd7d48

Will deploy it up soon—open to changes!
